cumulusci/cumulusci.yml

```diff
@@ -895,6 +895,24 @@ flows:
             2:
                 task: uninstall_managed
 
+    install_2gp_commit:
+        group: Install / Uninstall
+        description: Install the 2GP package for the current commit
+        steps:
+            1:
+                task: github_package_data
+                options:
+                    context: "Build Feature Test Package"
+            2:
+                flow: beta_dependencies
+                options:
+                    update_dependencies:
+                        dependencies: ^^github_package_data.dependencies
```

Comment:
Btw, there's an important subtlety in how this works that I want to point out. Our feature test packages are built without specifying any dependencies, which means that the `github_package_data` task will return an empty list of dependencies. Because that's falsy, `update_dependencies` will fall back to using the dependencies defined in project_config.project__dependencies, which is how the 1gp dependencies end up getting installed.

What this means is that specifying the option here is not really doing anything -- but it might in the rare case that we need to customize a project to build a feature test package without skipping validation (because then the 2gp package has to list dependencies, and `github_package_data` will find them, and this option will be truthy).